### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by TheMowgliMan** [AspectOSTeam/AspectOS#4] [Suggestions]
Please add the turbowarp packaged html files as releases here.

**Comment by AIGamesDeveloper** [comment]
I'll consider this. It might be great to let people know, but it will only be a patch

**Comment by TheMowgliMan** [comment]
It would allow download links on websites with the a tag

**Reply by TheMowgliMan** [reply]
Also I noticed that I have written more posts than most people do in a week in the last 24 hours

**Reply by TheMowgliMan** [reply]
I'm bossy

**Comment by Eastboat** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
